.idea/compiler.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
unnecessary file

Reply:
gitignore needs updating, will deal with it next week